src/views/Workload/ListView.vue

```diff
@@ -45,82 +46,79 @@
   import { useToast } from 'primevue/usetoast'
   import { INFORMATION_TEXTS, TEXT_DOMAIN_WORKLOAD } from '@/helpers'
   const handleTextDomainWorkload = TEXT_DOMAIN_WORKLOAD()
+  import { workloadService } from '@/services/v2'
+  import * as Helpers from '@/helpers'
 
   import PageHeadingBlock from '@/templates/page-heading-block'
   import { computed, ref, inject } from 'vue'
 
   /**@type {import('@/plugins/analytics/AnalyticsTrackerAdapter').AnalyticsTrackerAdapter} */
   const tracker = inject('tracker')
 
-  const props = defineProps({
-    listDomainsService: {
-      required: true,
-      type: Function
-    },
-    deleteDomainService: {
-      required: true,
-      type: Function
-    },
-    documentationService: {
-      required: true,
-      type: Function
-    },
-    clipboardWrite: {
-      required: true,
-      type: Function
-    }
-  })
-
   const createDomainPath = `${handleTextDomainWorkload.pluralLabel}/create?origin=list`
   const toast = useToast()
   const DOMAINS_API_FIELDS = [
-    'id',
     'name',
-    'edge_application',
+    'domains',
+    'workload_domain',
+    'infrastructure',
     'active',
-    'alternate_domains',
-    'workload_hostname',
-    'product_version'
+    'last_modified',
+    'id',
+    // 'protocols',
+    'last_editor',
+    'product_version',
+    'workload_domain'
   ]
 
+  const columnsHiddenByDefault = ['id', 'lastEditor', 'protocols']
+
   const hasContentToList = ref(true)
+
+  const deleteWorkloas = (id, name) => {
+    return {
+      id,
+      name
+    }
+  }
+
   const actions = [
     {
       type: 'delete',
       title: 'domain',
       icon: 'pi pi-trash',
-      service: props.deleteDomainService
+      service: deleteWorkloas
```

Comment:
Typo. Ajustar nos outros pontos necessários
```suggestion
      service: deleteWorkloads
```